lib/modules/manager/npm/extract/index.spec.ts

```diff
@@ -293,6 +293,28 @@ describe('modules/manager/npm/extract/index', () => {
       ).toBeArrayIncludingOnly(['https://registry.example.com']);
     });
 
+    it('resolves registry URLs using the package name if set', async () => {
+      fs.readLocalFile.mockImplementation((fileName): Promise<any> => {
```

Comment:
Why not simply use `mockResolvedValueOnce`?

Reply:
That's not equivalent is it? This implementation resolves to `null` for other filenames. It's also consistent with (copied from 😅) the existing tests.

Reply:
I wanted to know if you have tried it, because it should work with that too.